apps/package.json

```diff
@@ -8,8 +8,11 @@
     "npm": "^2.9.1"
   },
   "scripts": {
-    "build": "grunt build",
-    "test": "grunt build && grunt test"
+    "lint": "grunt jshint",
+    "test": "grunt build && grunt test",
```

Comment:
We're in the middle of that right now :)

On Wed, Feb 17, 2016 at 4:26 PM, Brent Van Minnen <notifications@github.com>
wrote:

> In apps/package.json
> <https://github.com/code-dot-org/code-dot-org/pull/6842#discussion_r53255122>
> :
>
> > @@ -8,8 +8,11 @@
> >      "npm": "^2.9.1"
> >    },
> >    "scripts": {
> > -    "build": "grunt build",
> > -    "test": "grunt build && grunt test"
> > +    "lint": "grunt jshint",
> > +    "test": "grunt build && grunt test",
>
> Can we make test not also build? Otherwise I'm never going to use npm test
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6842/files#r53255122>.
>



-- 
Brad Buchanan
Software Engineer, Code.org
brad@code.org